packages/opentelemetry-browser-extension-autoinjection/package.json

```diff
@@ -26,60 +27,62 @@
     "node": ">=14"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.0"
+    "@opentelemetry/api": "^1.2.0"
```

Comment:
Why upgrade the "@opentelemetry/api" minimum supported minor versions?

If the user is using an older version of the API, will it fail to work?

Reply:
Not sure why I changed it, it _should_ work with 1.0 as well